mm/kfence.c

```diff
@@ -18,10 +18,14 @@ struct stored_freelist {
 	struct kmem_cache *cache;
 	void *freelist;
 };
+
+enum kfence_object_state { KOS_UNUSED, KOS_ALLOCATED, KOS_FREED };
```

Comment:
KOS is a bit cryptic. Maybe just 'KFENCE_OBJECT_' ?

Reply:
Agreed. I thought it'll be used very often, but it doesn't seem to be the case.